flood_adapt/object_model/utils.py

```diff
@@ -22,3 +23,37 @@ def write_finished_file(path: Path):
 
 def finished_file_exists(path: Path):
     return (Path(path) / "finished.txt").exists()
+
+
+def import_external_file(
+    external_file: Path | str | os.PathLike, dst_dir: Path | str | os.PathLike
+) -> Path:
+    """Copy an external file to the destination directory.
+
+    Parameters
+    ----------
+    external_file : Path | str | os.PathLike
+        Path to the external file to be copied.
+    dst_dir : Path | str | os.PathLike
+        Path to the destination directory.
+
+    Returns
+    -------
+    Path
+        Path to the copied file.
+
+    Raises
+    ------
+    FileNotFoundError
+        If the external file does not exist.
+    """
+    external_file = Path(external_file).resolve()
+    dst_dir = Path(dst_dir).resolve()
+    if not external_file.exists():
+        raise FileNotFoundError(
+            f"Could not import file {external_file} as it does not exist."
+        )
+    dst_dir.mkdir(parents=True, exist_ok=True)
+    shutil.copy2(external_file, dst_dir / external_file.name)
+
+    return dst_dir / external_file.name
```

Comment:
This method should return only the external_file.name, right? Since we want the database to be transferable. Else, this returns the absolute path to the file.

Reply:
Correct, I didnt realize we wanted to support users sharing databases like that, but it makes sense. Will have a look tomorrow to see how to fix this nicely.